content/blog/crosswalk-for-kubernetes-intro/index.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Is this k8sx file here on purpose?

Reply:
Yes, but I'm guessing that single branch for just blog posts is not good practice?

Reply:
I think it's better to keep them separate